cfpb/cfgov-refresh
# Allow HMDA Explorer lender IDs to contain dashes

We use JS to allow only numbers to be entered into the Lender ID field. Unfortunately, lender ID numbers also contain a dash. This PR adds the dash key to the list of permitted keycodes for that field.

## Testing

1. Go to https://www.consumerfinance.gov/data-research/hmda/explore, change the filters to "all records", open the lender section and try entering a dash into the Lender ID field. It should prevent you from doing so.
1. Pull down this branch.
1. Go to http://0.0.0.0:8000/data-research/hmda/explore and try entering a dash into the Lender ID field. It should now allow it.

## Screenshots

Before:

<img width="984" alt="Screen Shot 2019-08-02 at 2 51 24 PM" src="https://user-images.githubusercontent.com/1060248/62392395-570ee980-b535-11e9-9809-946df10230c2.png">

After:

<img width="954" alt="Screen Shot 2019-08-02 at 2 51 43 PM" src="https://user-images.githubusercontent.com/1060248/62392698-1cf21780-b536-11e9-9278-b8256c44f41e.png">
